build/debs/BUILD

```diff
@@ -163,7 +164,7 @@ k8s_deb(
     description = """Kubernetes Packaging of CNI
 The Container Networking Interface tools for provisioning container networks.
 """,
-    version_file = "//build:os_package_version",
+    version_file = "//build:cni_package_version",
```

Comment:
just use version = 0.5.1 and skip the genrule.

Reply:
I wanted to use version_file, otherwise it will be most packages will be generated with names %name-%arch.rpm, and this one will be %name-%version-%arch.rpm. Later on, if we will fix that version_file will generate same filename as if it will use version argument, it will be ok to just put hardcoded version here.